DesignRules.md

```diff
@@ -446,6 +446,39 @@ Resources are often interconnected by relationships. Relationships can be modell
   <p></p>
 </div>
 
+## Conditional Modules
+
+As described in <a href="#extensions">the Extensions paragraph</a> the NL API Strategy is composed of a set of modules that are extentions on <a href="#the-core-set-of-design-rules">the Core set of Design Rules</a>. The modules are optional or conditional. Optional rules can be found in the infographic. All conditional modules are referenced in this document and the 'conditional usage' is formalized as a unique designrule.
+
+<span id="api-tls"></span>
+<div class="rule" id="/core/conditional/transport-security">
+  <p class="rulelab"><b>/core/conditional/transport-security</b>: Apply transport security conform the module</p>
```

Comment:
Is niet helemaal correct Engels. Misschien aanpassen naar: "Apply the transport security module"?

Reply:
deze heb ik verbeterd